src/xcode/ENA/ENA/Source/Scenes/TraceLocations/Configuration/TraceLocationConfigurationViewController.swift

```diff
@@ -154,6 +169,22 @@ class TraceLocationConfigurationViewController: UIViewController, FooterViewHand
 			startDatePicker.preferredDatePickerStyle = .inline
 			endDatePicker.preferredDatePickerStyle = .inline
 		}
+
+		startDatePicker.addTarget(self, action: #selector(selectedDatPicker(sender:)), for: .editingDidBegin)
+		endDatePicker.addTarget(self, action: #selector(selectedDatPicker(sender:)), for: .editingDidBegin)
+
+		startDatePicker.addTarget(self, action: #selector(deselectDatePicker(sender:)), for: .editingDidEnd)
+		endDatePicker.addTarget(self, action: #selector(deselectDatePicker(sender:)), for: .editingDidEnd)
+	}
+
+	@objc
+	private func selectedDatPicker(sender: UIDatePicker) {
```

Comment:
selectedDatePicker* or selectDatePicker to make consistent with deselectDatePicker